src/containers/Bookings.js

```diff
@@ -1,18 +1,62 @@
 import React, { Component } from "react";
 import Search from "../components/Search.js";
+import ResultsTable from '../components/ResultsTable'
```

Comment:
Do you use Windows? though this might work on your machine this file is named `resultsTable.js` and you import it with capital here. Best to always use capital in react projects